controls/srg_ctr/SRG-APP-000297-CTR-000705.yml

```diff
@@ -4,4 +4,6 @@ controls:
   - low
   title: Access to the container platform must display an explicit logout message
     to user indicating the reliable termination of authenticated communication sessions.
-  status: pending
+  status: automated
+  rules:
+    - oauth_logout_url_set
```

Comment:
Could you include an empty line at the end, please?

Reply:
done, thank you